dijkstra/Workspace_Map.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pygame as pg
+import cv2
+
+#pg.init()
+class drawMap:
+    def __init__(self,radius,clearance):
+        
+        self.R = radius
+        self.C = clearance
+        self.height = 200
+        self.width = 300
+        #self.Surface = pg.display.set_mode((self.width,self.height))
+        self.ax = self.R+self.C
+    
+    def getMap(self,window):
+        blue = (0,0,255)
```

Comment:
here, blue and white are like constant variables (their values are not going to change throughout the execution of the code): so for constants we can name the variables as 'all caps letters', eg: BLUE_COLOR/BLUE/BLUE_COLR and same goes for white and ther constant variables.

Please make the required change

Reply:
Done